python/triton/backends/__init__.py

```diff
@@ -1,10 +1,15 @@
 import importlib
 import inspect
-import os
+import sys
 from dataclasses import dataclass
 from .driver import DriverBase
 from .compiler import BaseBackend
 
+if sys.version_info >= (3, 10):
+    from importlib.metadata import entry_points
+else:
+    from importlib_metadata import entry_points
```

Comment:
What's the difference between if/else branches?

Reply:
Yeah, the former is a built-in module, and the latter is the backport.